### DISCUSSION TITLE
Entity lost

### DISCUSSION THREAD

**Discussion by Smk3416** [BottlecapDave/HomeAssistant-OctopusEnergy#944] [Q&A]
Suddenly lost all my electricity entities. Have become unavailable. Tried reloading/reinstalling and now it won't even connect to my smart meter, yet it picks up the export and gas meters fine

**Comment by BottlecapDave** [comment]
Hello and sorry that you're seeing this issue. Can you download the diagnostics for your account (details in the FAQ for this integration) and past the information here?

**Comment by Smk3416** [comment]
I would, but I reinstalled the integration. Now it won’t connect to the meter that would show the diagnostics. Unless there is any other way to pull up the information?


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

________________________________

**Reply by BottlecapDave** [reply]
You can download the diagnostics from any of the available meters, as they all return the same data.
